packages/shared/src/hooks/useActiveNav.tsx

```diff
@@ -29,6 +29,7 @@ export default function useActiveNav(activeFeed: AllFeedPages): UseActiveNav {
       OtherFeedPage.Tags,
       OtherFeedPage.Sources,
       OtherFeedPage.Leaderboard,
+      OtherFeedPage.Following,
```

Comment:
Maybe we can rename this file to be `useActiveNav.ts` instead of `useActiveNav.tsx`. We don't return any JSX as I can see?

Reply:
Doesn't change/harm anything right?

Reply:
Nah it shouldn't